x-pack/plugins/cases/public/common/lib/kibana/hooks.ts

```diff
@@ -167,9 +167,15 @@ export const useNavigation = (appId: string) => {
 export const useApplicationCapabilities = (): { crud: boolean; read: boolean } => {
   const capabilities = useKibana().services.application.capabilities;
   const casesCapabilities = capabilities[FEATURE_ID];
-
   return {
     crud: !!casesCapabilities?.crud_cases,
     read: !!casesCapabilities?.read_cases,
   };
 };
+export const useKibanaCapabilities = (): { visualize: boolean } => {
+  const capabilities = useKibana().services.application.capabilities;
+
+  return {
+    visualize: !!capabilities?.visualize?.save,
```

Comment:
Currently it seems you also need `save` on the Dashboard feature privilege too. Without that, you end up in Lens, but no way to `Save and Return` to Cases as that button is disabled.

![image](https://user-images.githubusercontent.com/7405507/158370787-cbc5b032-0188-4d66-bf4d-5e7796f465b5.png)